tests/run-travis.sh

```diff
@@ -11,14 +21,14 @@ function run_packages_tests {
 			export NAME=$(basename $(pwd))
 
 			if [ ! -e tests/php/travis-can-run.sh ] || tests/php/travis-can-run.sh; then
-				if [ "$DO_COVERAGE" == "true" ]; then
-					composer install
+				if [ "$DO_COVERAGE" == "true" ] && [[ "$NAME" != "codesniffer" ]]; then
+					composer install -q
```

Comment:
In past debugging I've found it useful to see what `composer install` has to output...

Reply:
removed `-q` in 4000664